### ISSUE TITLE
The difference between hard keyboard and soft keyboard(Google Pixel Phone)

### ISSUE THREAD

**Issue by jakekwak** [Vanessa219/vditor#434]
I trying to debug my pixel phone using USB Debug mode.
I found one thing. (the starting time of `compositionstart` and `isComposing` is `true` when soft keyboard)

Hard keyboard
![vditor45](https://user-images.githubusercontent.com/15811943/82744580-260bd980-9d40-11ea-9a87-e2e87e657c68.gif)

![vditor47](https://user-images.githubusercontent.com/15811943/82744640-f3161580-9d40-11ea-86a6-110fc87de9f7.gif)

Soft keyboard(on screen, Gboard)

![vditor46](https://user-images.githubusercontent.com/15811943/82744586-3328c880-9d40-11ea-8501-de452f3fef97.gif)

![vditor48](https://user-images.githubusercontent.com/15811943/82744642-f6a99c80-9d40-11ea-8088-59797f16a897.gif)

**Comment by jakekwak** [comment]
Gboard(google soft keyboard)'s backspace returned `unidentified` key not `backspace`
![71](https://user-images.githubusercontent.com/15811943/82760535-b33f4480-9db9-11ea-8dc0-d8d74797bbbd.PNG)

**Comment by Vanessa219** [comment]
So, I can not resolve this problem?

**Comment by jakekwak** [comment]
I reported this issue to google. (I am using the different soft keyboard).
and I had another iPhone(iPad) issue. ( I also reported this issue to apple.)
no composition event at Korean.
![a3](https://user-images.githubusercontent.com/15811943/82913395-da218600-9f33-11ea-8fb3-dc522637d5f6.png)

Chinese OK.(Japanese OK)
![a1](https://user-images.githubusercontent.com/15811943/82913427-e60d4800-9f33-11ea-8ea1-ba334afbc31c.png)
